pyccl/tests/test_neutrinos.py

```diff
@@ -18,7 +18,8 @@ def test_omnuh2_smoke(a, m):
     assert np.shape(om) == np.shape(a)
 
 
-@pytest.mark.parametrize('split', ['normal', 'inverted', 'equal', 'sum'])
+@pytest.mark.parametrize('split', ['normal', 'inverted', 'equal', 'sum',
+                         'single'])
```

Comment:
tiny style note, but if you want to wrap this to the next line, it's normally better to start the next line below the relevant enclosing bracket or paren.  So here it would be:
```
@pytest.mark.parametrize('split', ['normal', 'inverted', 'equal', 'sum',
                                   'single'])
```
or
```
@pytest.mark.parametrize('split', 
                         ['normal', 'inverted', 'equal', 'sum', 'single'])
```

Reply:
flake8 will demand such things, which alleviates the need for us to comment on them. ;p

j/k j/k but flake8 here will choke

Reply:
Fixing now, good point.